Mailer/app/api/v1/Handlers/Users/SubscribeHandler.php

```diff
@@ -2,16 +2,16 @@
 
 namespace Remp\MailerModule\Api\v1\Handlers\Users;
 
+
 use Nette\Database\Table\ActiveRow;
-use Nette\Utils\Json;
-use Nette\Utils\JsonException;
 use Remp\MailerModule\Api\InvalidApiInputParamException;
+use Remp\MailerModule\Api\v1\Handlers\Validation\InputValidatorTrait;
```

Comment:
InputValidationTrait doesn't need to be under `v1` folder/namespace, it's generic enough to be within `Remp\MailerModule\Api` namespace.